web/src/queries/software.ts

```diff
@@ -143,6 +145,14 @@ const repositoriesQuery = () => ({
   queryFn: fetchRepositories,
 });
 
+/**
+ * Query to retrieve conflicts
+ */
+const conflictsQuery = () => ({
+  queryKey: ["software/conflicts"],
```

Comment:
As said in at https://github.com/agama-project/agama/pull/2230#discussion_r2036883061, although we still having to update some parts in the code, 

> We prefer using keys in the format of ["software", "registration", "addons"]. It is a better practice for cleaning caches, sorry if you found that format, we haven't had time to fix them.
> 
> See https://tanstack.com/query/v5/docs/framework/react/guides/query-keys and https://tkdodo.eu/blog/effective-react-query-keys